apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/oversendelsebrev/OversendelseBrevService.kt

```diff
@@ -189,13 +194,30 @@ class OversendelseBrevServiceImpl(
                     id = brev.id,
                     bruker = brukerTokenInfo,
                     automatiskMigreringRequest = null,
-                    avsenderRequest = { bruker, generellData -> AvsenderRequest(bruker.ident(), generellData.sak.enhet) },
+                    avsenderRequest = { bruker, generellData ->
+                        AvsenderRequest(
+                            bruker.ident(),
+                            generellData.sak.enhet,
+                        )
+                    },
                     brevKode = { Brevkoder.OVERSENDELSE_KLAGE },
                     brevData = { req -> OversendelseBrevFerdigstillingData.fra(req, klage) },
                 )
             }
         return pdf
     }
+
+    override suspend fun slettOversendelseBrev(
+        behandlingId: UUID,
+        brukerTokenInfo: BrukerTokenInfo,
+    ) {
+        val eksisterendeBrev =
+            brevRepository.hentBrevForBehandling(behandlingId, Brevtype.OVERSENDELSE_KLAGE)
+                .singleOrNull()
+        if (eksisterendeBrev != null) {
```

Comment:
Greit å ha en sjekk på at brevet ikke er journalført / distribuert her først, så det ikke forsvinner fra brevoversikten mens det lever i dokumentoversikten

Reply:
Ja, det er sant. Her bør det vel være likt som for sletting av vedtaksbrev. Er vel bare brevtypen som er forskjellig.